src/transactions/MergeTests.cpp

```diff
@@ -127,7 +150,9 @@ TEST_CASE("merge", "[tx][merge]")
             REQUIRE(a1.getBalance() ==
                     a1Balance + b1Balance - txFrame->getFee());
             REQUIRE(!loadAccount(b1, *app, false));
-            REQUIRE(a1SeqNum == a1.loadSequenceNumber());
+            // a1 gets recreated with a sequence number based on the current
+            // ledger
+            REQUIRE(a1.loadSequenceNumber() == 0x300000000ull);
```

Comment:
Won't it break catchup complete on current pubnet?

Reply:
no it won't: what I did here is change the test to be explicit about what is going on by adding an extra `    closeLedgerOn(*app, 2, 1, 1, 2016); ` after setting up the original accounts.
the check `REQUIRE(a1SeqNum == a1.loadSequenceNumber())` was just misleading because the reason `a1SeqNum` was not changing was that the setup and the transaction being tested (with all its sub-operations) were occurring during the same ledger.